pom.xml

```diff
@@ -93,6 +93,12 @@
 			<version>${junitVersion}</version>
 			<scope>test</scope>
 		</dependency>
+                <dependency>
+                        <groupId>org.junit.jupiter</groupId>
+                        <artifactId>junit-jupiter-engine</artifactId>
+                        <version>${junitVersion}</version>
+                        <scope>test</scope>
+                </dependency>
```

Comment:
It does not seems to be related to this PR topic.

Reply:
The pom.xml change yes, but I think the other changes are actually related and pretty much required, no? Otherwise by creating a QuantityIdValue from the Datamodel helper, you will first convert an ItemIdValue to a String and then back to an ItemIdValue.

Reply:
Yes, indeed. I was mentioning this pom.xml change.